server/etcdserver/apply/apply.go

```diff
@@ -233,14 +232,14 @@ func (a *applierV3backend) Alarm(ar *pb.AlarmRequest) (*pb.AlarmResponse, error)
 			break
 		}
 		resp.Alarms = append(resp.Alarms, m)
-		alarms.With(prometheus.Labels{"server_id": strconv.FormatUint(ar.MemberID, 10)}).Inc()
+		alarms.With(prometheus.Labels{"server_id": fmt.Sprintf("%016x", ar.MemberID)}).Inc()
```

Comment:
Please cast it to [types.ID](https://github.com/etcd-io/etcd/blob/53bfdc11d8f6d6cdf0ccb986f052e9739086110b/client/pkg/types/id.go#L22-L24)